packages/core/package.json

```diff
@@ -2,31 +2,24 @@
   "name": "@clr/core",
   "version": "4.0.0",
   "scripts": {
-    "start": "npm-run-all build -p build:watch storybook",
-    "storybook": "start-storybook -s .storybook/public -p 6006 --config-dir=.storybook",
-    "dev:build": "build-storybook -s .storybook/public -c .storybook -o ../../dist/website/storybook/core --quiet",
+    "start": "npm-run-all build -p build:watch storybook:start",
+    "storybook:start": "start-storybook -s .storybook/public -p 6006 --config-dir=.storybook",
+    "storybook:build": "build-storybook -s .storybook/public -c .storybook -o ../../dist/website/storybook/core --quiet",
     "lint": "eslint \"**/*.ts\" --ignore-pattern \"dist\" && stylelint \"./**/*.scss\" && lit-analyzer ./src",
     "lint:fix": "eslint --fix \"./**/*.ts\" && stylelint --fix \"./**/*.scss\"",
-    "golden": "yarn build:api",
-    "golden:fix": "yarn golden && cpy ./../dist/core/custom-elements.json ./",
-    "golden:test": "diff -u ./custom-elements.json ./dist/core/custom-elements.json",
```

Comment:
These are not being used and the API output is not reliable format yet. Once stable we can enable golden file check on core